docs/de/docs/tutorial/first-steps.md

```diff
@@ -0,0 +1,335 @@
+# Erste Schritte
+
+Die einfachste FastAPI-Datei könnte wie folgt aussehen:
+
+```Python
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Kopieren Sie diese in die Datei `main.py`.
+
+Starten Sie den Live-Server:
+
+<div class="termy">
+
+```console
+$ uvicorn main:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+<span style="color: green;">INFO</span>:     Started reloader process [28720]
+<span style="color: green;">INFO</span>:     Started server process [28722]
+<span style="color: green;">INFO</span>:     Waiting for application startup.
+<span style="color: green;">INFO</span>:     Application startup complete.
+```
+
+</div>
+
+!!! Hinweis
+    Der Befehl `uvicorn main:app` bezieht sich auf:
+
+    * `main`: die Datei `main.py` (das Python "Modul").
+    * `app`: das Objekt, das innerhalb von `main.py` mit der Zeile `app = FastAPI()` erzeugt wird.
+    * `--reload`: lässt den Server nach Codeänderungen neu starten. Nur für die Entwicklung zu verwenden.
+
+In der Ausgabe sieht man eine Zeile wie diese:
+
+```hl_lines="4"
+INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+In dieser Zeile wird die URL angezeigt, unter der Ihre Anwendung auf Ihrem lokalen Rechner bereitgestellt wird.
+
+### Probieren Sie es aus
+
+Öffnen Sie ihren Browser unter der Addresse <a href="http://127.0.0.1:8000" class="external-link" target="_blank">http://127.0.0.1:8000</a>.
+
+Sie bekommen die JSON-Antwort als:
+
+```JSON
+{"message": "Hello World"}
+```
+
+### Interaktive API-Dokumentation
+
+Jetzt gehen Sie auf <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
+
+Es wird die automatische interaktive API-Dokumentation angezeigt (zur Verfügung gestellt von <a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>):
+
+![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
+
+### Alternative API-Dokumentation
+
+Gehen Sie jetzt auf <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>.
+
+Jetzt wird die alternative automatische Dokumentation angezeigt (zur Verfügung gestellt von <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
+
+### OpenAPI
+
+**FastAPI** generiert ein "Schema" für Ihre gesamte APIs unter Verwendung des **OpenAPI**-Standards zur Definition von APIs.
```

Comment:
"gesamten" APIs (plural)

Reply:
Actually, I check the englisch version again and its singular. So probably `gesamte API`